lib/run/index.js

```diff
@@ -133,10 +133,33 @@ module.exports = function (options, callback) {
                 result[value] = true;
             }, {});
         }
+        let RequiredFolder = [];
 
+        function checkItems (item) {
```

Comment:
Is this okay to acculturate all folders at the start of the run always? What will happen memory for collections with hundreds of folders?